library/src/main/java/com/novoda/downloadmanager/lib/DownloadManager.java

```diff
@@ -443,8 +443,40 @@ public int markRowDeleted(long... ids) {
      * @param ids the IDs of the downloads to remove
      * @return the number of downloads actually removed
      */
-    public int remove(long... ids) {
-        return markRowDeleted(ids);
+    public int removeDownloads(long... ids) {
```

Comment:
Yeah this is weird right? We had that conversation as well and there are two irrelevant public methods there that just redirects.

If noone sees a reason for those methods, we'll more than happy to delete them.